base/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerinax"
 name = "health.hl7v2"
-version = "1.0.5"
```

Comment:
Why are we directly moving to the major version 2.0.0?

Reply:
we're introducing a change that's backward-incompatible.